cmd/e2e-test/main.go

```diff
@@ -484,7 +484,7 @@ func createProcess(
 				if err := json.Unmarshal(pBytes, proc); err != nil {
 					return nil, nil, fmt.Errorf("failed to unmarshal process response: %v", err)
 				}
-				processReady = proc.IsAcceptingVotes
+				processReady = proc.IsAcceptingVotes && proc.IsLocallyActive
```

Comment:
IsAcceptingVotes and IsLocallyActive seems redundant to me. Isn't it possible to just use one of them?

Reply:
I'm not sure about the relation between "a sequencer do not have encryption keys" and "is locally active". 

Not having encryption keys do not mean a sequencer cannot process votes and send batches. 

The logic behind `isAcceptingVotes` makes sense to me. A Sequencer might not want to participate in all voting processes or there might be some missing data (such as the current state) which invalidates the sequencer.